### ISSUE TITLE
i need pacp

### ISSUE THREAD

**Issue by M0he** [KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset#1]
Can you give the link about cic2018_datdaset(pcap) detail?

**Comment by KarenPamelaLopezMedina** [comment]
Of course
https://www.kaggle.com/datasets/karenp/cse-cic-ids2018
Of course, I uploaded the csv files to kaggle, this is the link, in the right corner you can click on download.
Any questions let me know.

**Comment by M0he** [comment]
"I want the original pcap file, mainly related to Botnet. Do you have a download link for it?"

**Comment by KarenPamelaLopezMedina** [comment]
Do you mean the files that are not preprocessed?
 I don't have them, I was looking for them but I delete them because they take up a lot of space on my computer. 
Do you need them urgently? If you like I try to download it again but last time it took me 2 days to download let me know

**Comment by M0he** [comment]
Yes,I need this data and   can you  help download the botnet part of IDS2018? Thank you very much.

**Comment by KarenPamelaLopezMedina** [comment]
I am trying to download again, the first time i remeber it was a little difficult for me (I am not an expert) let me check and in 2 hours I will write you again to inform you if I could begin the download, is it ok?

**Comment by M0he** [comment]
OK, thank you very much

**Comment by KarenPamelaLopezMedina** [comment]
![Captura de pantalla 2023-05-31 202707](https://github.com/KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset/assets/68719546/64c7b1e3-7818-4178-abe7-62cc14cb44b0)

Look!! I could start the download, we only have to wait until Friday...
We can be in touch during these days, letting us know that you still interested and I will show how it is advancing. 
It is ok for you?

**Comment by M0he** [comment]
OK，I am very willing to take the time to wait and thank you very much for your help

**Comment by KarenPamelaLopezMedina** [comment]
![Captura de pantalla 2023-06-01 070518](https://github.com/KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset/assets/68719546/47407ea9-001a-4f3b-b114-3bb42927a74b)
![Captura de pantalla 2023-06-01 070518](https://github.com/KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset/assets/68719546/47407ea9-001a-4f3b-b114-3bb42927a74b)


I don't know why the download failed, I'm downloading again, I'll let you know in 2 hours how it goes.

**Comment by M0he** [comment]
OK ,thanks.

**Comment by KarenPamelaLopezMedina** [comment]
![Captura de pantalla 2023-06-01 100633](https://github.com/KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset/assets/68719546/76ae4edc-308d-46df-85c4-ef779be9faa6)
 Everything is going well!

**Comment by KarenPamelaLopezMedina** [comment]
Hi, I have already downloaded, only miss to upload and I will send you the link!
Do you continue reading me?

**Comment by M0he** [comment]
Yeah，I need  it,thank for your help!

**Comment by KarenPamelaLopezMedina** [comment]
I have it I am thinking how to upload, see 446 GB.
![Captura de pantalla 2023-06-03 210147](https://github.com/KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset/assets/68719546/876d1330-2c86-4bc1-a738-60c30fc03412)
![Captura de pantalla 2023-06-03 210100](https://github.com/KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset/assets/68719546/0458c9e5-e7f9-4602-ad1c-0eb34ee178d6)

Do you have an idea?

**Comment by M0he** [comment]
![image](https://github.com/KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset/assets/126653631/1d385be4-996e-4e92-8874-89356b8a3172)
 I just need this pacp about botnet attack,can you upload to your kaggle?

**Comment by KarenPamelaLopezMedina** [comment]
I have not ever use this dataset, I'm trying to upload Friday-02-03-2018 pacp. Just wait
![Captura de pantalla 2023-06-04 074506](https://github.com/KarenPamelaLopezMedina/csv_csE-CIC-IDS2018_dataset/assets/68719546/9cdba7b5-b13b-4d09-b021-abb0672cdd77)

**Comment by KarenPamelaLopezMedina** [comment]
Hi @M0he  finnaly it's here :D I will upload the rest.
https://www.kaggle.com/datasets/karenp/original-network-traffic-friday-02-03-2018-pacp

**Comment by M0he** [comment]
Thank you very much，I am working on the characteristics of network behavior based on remote access Trojans，so i need botnat pcap.Thank you again for helping me

**Comment by M0he** [comment]
I believe you will help more researchers in the future.

**Comment by KarenPamelaLopezMedina** [comment]
It was a pleasure to help you.
Thanks for talking about what you're working on.
I am studying my doctorate in computer science and I work in classification in data stream, I was thinking of using this dataset but so far I don't think I will use it. Thank you very much for writing to me again. I also hope I can continue to be useful to more people and it was nice meeting you.
